packages/schemas/src/constants.ts

```diff
@@ -1,2 +1,2 @@
 export const DEFAULT_OPACITY = 1;
-export const HEX_COLOR_PATTERN = '^#(?:[A-Fa-f0-9]{3,4}|[A-Fa-f0-9]{6}|[A-Fa-f0-9]{8})$';
+export const HEX_COLOR_PATTERN = '^#(?:[A-Fa-f0-9]{6})$';
```

Comment:
the renderer colour formatting only accepts full 6 char hex codes, nothing else works (currently)